common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -74,6 +75,18 @@ class DB(ctx: JdbcContext[PostgresDialect, SnakeCase]) {
     }
   }
 
+  def insertImages(images: List[Image]): Unit = {
```

Comment:
Can this take a Seq[Image] so you don't have to do .toList further down when calling the function?